docs/Overview.rst

```diff
@@ -36,17 +36,67 @@ MVS Validation Plan
 
-*Conceptual model validation* consists of looking into the underlying theories and assumptions. Therefore, the conceptual validation scheme includes a comprehensive review of the generated equations by the oemof python library and the components’ models. Next step is to try and adapt them to one pilot project with specific constraints. Tracing and examining the flowchart is also considered as part of this validation type. The aim is to assess the reasonability of the model behavior through pre-requisite knowledge. 
+*Conceptual model validation* consists of looking into the underlying theories and assumptions. Therefore, the conceptual validation scheme includes a comprehensive review of the generated equations by the oemof-solph python library and the components’ models. Next step is to try and adapt them to one pilot project with specific constraints. Tracing and examining the flowchart is also considered as part of this validation type. The aim is to assess the reasonability of the model behavior through pre-requisite knowledge; this technique is known as face validity. 
 
-*Model verification* is related to computer programming and looks into whether the code is a correct representation of the conceptual model. To accomplish that, integration tests for each module will be written to assert that the output is as expected. The simulation will also be run several times for the same input data to double check the results. 
+*Model verification* is related to computer programming and looks into whether the code is a correct representation of the conceptual model. To accomplish this, static testing methods are used to validate the output with respect to an input. Unit tests and integration tests, using proof of correctness techniques, are integrated within the code and evaluate the output of the MVS for any change occuring as they are automated. Unit tests target a single unit such as an individual component, while integration tests target more general parts such as entire modules. Finally, it is very important to make sure that Pytest, the chosen framework for functional testing, always goes through.
 
-*Operational validity* determines if the model’s output is within the required accuracy. In order to achieve that, several validation techniques are used, namely:
+*Operational validity* assesses the model’s output with respect to the required accuracy. In order to achieve that, several validation techniques are used, namely:
 
-*	Benchmark testing, through which scenarios are created with different constraints and component combinations, and the output is calculated and compared to the expected one;
+* Graphical display, which is the use of model generated or own graphs for result interpretation. Graphs are simultaneously used with other validation techniques to inspect the results;
```

Comment:
```suggestion
* **Graphical display**, which is the use of model generated or own graphs for result interpretation. Graphs are simultaneously used with other validation techniques to inspect the results
```
Where is that done? If you did not add a paragraph on this, you could also update a link to your thesis to [here](https://multi-vector-simulator--593.org.readthedocs.build/en/593/References.html), and link to that source.

Reply:
Maybe you can mention that some of these displays can also be checked as benchmark tests automatically?

Reply:
I thought saying that it's used simultaneously with other validation techniques is enough.

Reply:
I added a sentence in sensitivity analysis that points back to that point